test/keyboardNavigation.test.tsx

```diff
@@ -46,43 +51,25 @@ test('basic keyboard navigation', () => {
   userEvent.keyboard('{arrowleft}');
   validateCellPosition(0, 0);
 
-  // page {up,down}/home/end navigation
+  // page {up,down}
   fireEvent.keyDown(document.activeElement!, { key: 'PageDown' });
```

Comment:
https://testing-library.com/docs/ecosystem-user-event/#keyboardtext-options
This should work I believe
```ts
userEvent.keyboard('{PageDown}');
```

Reply:
Did not realize `{shift}` and `{Shift}` are different